### DISCUSSION TITLE
is this project free if i  self hosted?

### DISCUSSION THREAD

**Discussion by ivan-vilches** [supabase/supabase#10181] [General]
is this project free on self hosted? if yes, self hosting have limitations ? can be used for commercial apps for free self hosted ?
thanks you very much.

**Comment by GaryAustin1** [comment]
It is free.  The main limitation is the studio UI is missing a few features on the hosted dashboard and does not manage projects.  Right now realtime v2 is not working with the docker/cli version but that is being worked on.

https://github.com/supabase/supabase/blob/master/LICENSE

**Reply by jameswong3388** [reply]
> It is free. The main limitation is the studio UI is missing a few features on the hosted dashboard and does not manage projects. Right now realtime v2 is not working with the docker/cli version but that is being worked on.
> 
> https://github.com/supabase/supabase/blob/master/LICENSE

Hi @GaryAustin1, where can I refer to regarding this issues **Right now realtime v2 is not working with the docker/cli version but that is being worked on.**
